package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "types-roll20",
-  "version": "1.1.2",
+  "version": "2.0.0",
```

Comment:
I'm not using the `package.json` `version`, but the base number from `version.json` file + an auto-incremented patch number based on the number of git commit since the last version bump. See [Nerdbank.GitVersioning](https://github.com/dotnet/Nerdbank.GitVersioning/) for more info.

Reply:
Please increment the `version.json` one.

Reply:
I think this technically has breaking changes so a major change increment would make more sense would it not? Some times I might import previously won't exist anymore

Reply:
It is a breaking change, indeed, but the version is in `version.json` not `package.json` please bump it to 3.0 there (or both).